universitat-bremen-institut-fur-politikwissenschaft.csl

```diff
@@ -116,10 +132,15 @@
   </macro>
   <macro name="locator">
     <choose>
-      <if type="article-journal">
-        <text variable="volume" suffix=" "/>
-        <text variable="issue" prefix="(" suffix=")"/>
+      <if type="article-journal" variable="DOI">
+        <text variable="DOI" prefix="DOI: "/>
       </if>
+      <else-if type="article" match="any">
```

Comment:
are you sure this is right? Don't you want `type="article-journal"` here? For what (Mendeley I assume?) item type do you want this?